### ISSUE TITLE
delay start until HA is ready

### ISSUE THREAD

**Issue by helto4real** [net-daemon/netdaemon#184]
## Example code
```c#
// Insert any example code here that will help describe your requests

```

## Additional information

> Added by Discord user helto

**Comment by ludeeus** [comment]
How do you differ from "HA not started" to "User put in the wrong config"?

**Comment by helto4real** [comment]
@ludeeus have no idea right now haha

**Comment by helto4real** [comment]
Fixed in 0.15.0
